lib/easybill-rest-client/generic_api.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+module EasybillRestClient
+  module GenericApi
+    def self.new(resource_name, resource_class)
```

Comment:
[Simplified it.](https://github.com/ad2games/easybill-rest-client/pull/4/commits/fd271206de3d2ff0f8a2f0f86afd316da6380079?w=1)